src/data/testimonials.json

```diff
@@ -77,13 +119,6 @@
       "imageLink": "https://avatars.githubusercontent.com/themousepotato",
       "blogLink": "https://medium.com/@themousepotato/kharagpur-winter-of-code-2017-6e350c2252c7"
     },
-    {
-      "name": "Sayan Sinha",
-      "role": "KWoC '16 Student",
-      "quote": "KWoC was a lot more than experience and learning. Now, I had worked on the repositories and had interacted with their makers.",
-      "imageLink": "https://avatars2.githubusercontent.com/u/23132358?s=460&v=4",
-      "blogLink": "https://blog.kossiitkgp.org/kharagpur-winter-of-code-c4eeb4af71f1"
```

Comment:
This link is working instead - https://medium.com/kharagpur-open-source-society/kharagpur-winter-of-code-c4eeb4af71f1

Sayan, Yash and Shivam are our advisors, while Athitya was a valuable part of MetaKGP. I think we can keep their feedback if the links are working.

Reply:
Will update Sayan's link and add the Advisors' testimonials. 
Shivam's link is broken too, is there a working link for it?